public/js/login.js

```diff
@@ -0,0 +1,37 @@
+const email = document.getElementById('email');
+const password = document.getElementById('password');
+const btnLogin = document.getElementById('btn_login');
+const msg = document.getElementById('msg');
```

Comment:
use descriptive variable name like 
```suggestion
const errorLabel = document.getElementById('msg');
OR
const errorMsg = document.getElementById('msg');

const valiadationLabel = document.getElementById('msg');
```